server/src/modules/persisters/comments-persisters.ts

```
@@ -0,0 +1,193 @@
+import { Context } from '../../config/context';
+import { BaseModel } from '../../database/base-model';
+import DatabaseComments, { DatabaseCommentsInitializer, DatabaseCommentsId } from '../../types/database/DatabaseComments';
+import { DatabaseProfileDraftsId } from '../../types/database/DatabaseProfileDrafts';
+import { DatabaseUsersId } from '../../types/database/DatabaseUsers';
+
+export interface CommentsModel extends DatabaseComments {}
+
+export class CommentsModel extends BaseModel {
```

Biome (1.9.4)

[error] This class is unsafely merged with an interface.

The interface is declared here.

The TypeScript compiler doesn't check whether properties defined in the interface are initialized in the class.

(lint/suspicious/noUnsafeDeclarationMerging)